src/thunderbots/software/test/geom/util.cpp

```diff
@@ -254,7 +254,7 @@ TEST(GeomUtilTest, test_angle_sweep_circles)
     obs.push_back(Point(9, 10));
 
     std::pair<Point, Angle> testpair =
-        angleSweepCircles(Point(0, 0), Point(10, 10), Point(-10, 10), obs, 1.0);
+        *angleSweepCircles(Point(0, 0), Point(10, 10), Point(-10, 10), obs, 1.0);
```

Comment:
You should check the optional first like you do in other tests

Reply:
changed to:
```
    testpair_opt = angleSweepCircles(Point(0, 0), Point(10, 10), Point(-10, 10), obs, 1.0);

    // We expect to get a result
    ASSERT_TRUE(testpair_opt);

    testpair = *testpair_opt;
```